logic/src/commonMain/kotlin/com/wire/kalium/logic/data/team/TeamRepository.kt

```diff
@@ -168,13 +169,15 @@ internal class TeamDataSource(
     // TODO: there is no documentation about getting the next page in case there is a next page
     override suspend fun syncServices(teamId: TeamId): Either<CoreFailure, Unit> = wrapApiRequest {
         teamsApi.whiteListedServices(teamId = teamId.value)
-    }.map {
-        it.services.map { service ->
+    }.fold({
+        Either.Right(Unit)
```

Comment:
in this case this function will never fail, i suggest changing this in the. `SyncSelfTeamUseCase`
something like:

```kotlin
internal class SyncSelfTeamUseCaseImpl(
    private val userRepository: UserRepository,
    private val teamRepository: TeamRepository
) : SyncSelfTeamUseCase {

    override suspend fun invoke(): Either<CoreFailure, Unit> {
        val user = userRepository.observeSelfUser().first()

        return user.teamId?.let { teamId ->
            teamRepository.fetchTeamById(teamId = teamId).flatMap {
                teamRepository.fetchMembersByTeamId(
                    teamId = teamId,
                    userDomain = user.id.domain
                )
            }.onSuccess {
                teamRepository.syncServices(teamId = teamId)
            }
        } ?: run {
            kaliumLogger.withFeatureId(SYNC).i("Skipping team sync because user doesn't belong to a team")
            Either.Right(Unit)
        }
    }
}
```

Reply:
this way errors from syncing services will be ignored during sync